etl/pypasar/omop/note.py

```diff
@@ -22,15 +21,125 @@ def execute(self):
-        pass
+        # Set SCHEMA
+        omop_schema = os.getenv("POSTGRES_OMOP_SCHEMA")
+        # source_schema = os.getenv("POSTGRES_SOURCE_SCHEMA")
+
+        with self.engine.connect() as connection:
+            with connection.begin():
+                # Read from source and create a staging table (stg__note)
+                # create a staging table to perform transformation and joining other required tables
+                connection.execute(
+                    text(f'''
+                        CREATE OR REPLACE VIEW {omop_schema}.stg__note AS
+                            WITH postop__clindoc AS (
+                                SELECT
+                                    id,
+                                    anon_case_no,
+                                    session_id,
+                                    postop_clindoc_created_datetime,
+                                    postop_clindoc_item_name,
+                                    postop_clindoc_item_description,
+                                    postop_clindoc_value_text,
+                                    ROW_NUMBER() OVER (
+                                        PARTITION BY anon_case_no, session_id, postop_clindoc_created_datetime, postop_clindoc_item_description
+                                        ORDER BY id
+                                    ) AS row_num      --Any rows that are duplicate with show 1,2,...
+                                FROM postop.clindoc
+                            ),
+
+                            -- Filter the table to only include unique value
+                            filtered AS (
+                                SELECT * from postop__clindoc WHERE ROW_NUM = 1
+                            ),
+
+                            -- revert visit occurrence id to session id
+                            sessionIDs AS (
+                                SELECT CAST(LEFT(CAST(visit_occurrence_id AS TEXT), LENGTH(CAST(visit_occurrence_id AS TEXT)) - 2) AS INTEGER) AS session_id, *
+                                    FROM omop_sqldev_schema.visit_occurrence
+                            )
+
+                            SELECT
+                                ROW_NUMBER() OVER (ORDER BY clindoc.postop_clindoc_created_datetime, id) AS note_id,
+                                CDM_PER.person_id AS person_id,
+                                CAST(clindoc.postop_clindoc_created_datetime AS date) AS note_date, -- get date only
+                                clindoc.postop_clindoc_created_datetime AS note_datetime,
+                                clindoc.postop_clindoc_item_name AS note_title,
+
+                            -- Handle any null values in note text
+                            CASE
+                                WHEN clindoc.postop_clindoc_value_text IS NOT NULL 
+                                THEN clindoc.postop_clindoc_value_text
+                                ELSE '0'
+                            END AS note_text,
```

Comment:
@chowmunchun Since `note_text` is a varchar datatype and not integer, I would recommend to set it as null. Only for concept id columns which are of type integer are recommended by OHDSI (especially standard concept id columns) to mark them as `0`.